packages/runtime/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @openfn/runtime
 
+## 0.2.3
+
+### Patch Changes
+
+- f228fd5: Add support for initial edge conditions in worker and add edge evaluation logging to runtime
```

Comment:
This should be two different patch notes - changes to the worker should not be reflected in the runtime's log.

I'm happy to jump in and change this once we agree on the actual logging. This comment is kinda just a reminder to me.